cfp.yml

```diff
@@ -0,0 +1,21 @@
+"PLDI'25":
+  - url: "[PLDI'25](https://conf.researchr.org/series/pldi)"
+  - publisher: ACM
+  - CORE: "[A*](https://portal.core.edu.au/conf-ranks/84/)"
```

Comment:
@uchitsa it would be more elegant to name them all in small letters: `core`, `scope`, `short`, `full`, `cfp`, `country, etc.

Reply:
updated